File: 8kyu/(8 kyu) Jenny's secret message/(8 kyu) Jenny's secret message.py
# #1
# def greet(name):
#     if name == "Johnny":
#         return "Hello, my love!"
#     return "Hello, {name}!".format(name=name)
# #2
# def greet(name):
#     return "Hello, {name}!".format(name="my love" if name == "Johnny" else name)
# #3


def greet(n): return "Hello, {}!".format(n.replace("Johnny", "my love"))

File: 8kyu/(8 kyu) Jenny's secret message/(8 kyu) Jenny's secret message.cpp
#include <string>

// #1
// std::string greet(std::string name)
// {
//     return (name == "Johnny")
//                ? "Hello, my love!"
//                : "Hello, " + name + "!";
// }
// #2
std::string greet(std::string name)
{
    return "Hello, " + (name == "Johnny" ? "my love" : name) + "!";
}